addons/website_sale_loyalty/static/tests/tours/website_sale_delivery_gift_card.js

```diff
@@ -58,3 +58,47 @@ registry.category("web_tour.tours").add('shop_sale_loyalty_delivery', {
         },
     ]
 });
+
+registry.category("web_tour.tours").add('check_shipping_discount', {
```

Comment:
You are testing a promotion not a gift card, maybe worth to do it in a separate file?